### PR TITLE
Gatekeeper upgrade

### PR DESCRIPTION
**Pull request by 7navyasa**
### What does this PR do?
Upgrading gatekeeper example claims, template and constraints

### Motivation

- Example claims not working 
- Creating gatekeeper workshop

### More

- [x] Yes, I have tested the PR using my local account setup  (Provide any test evidence report under Additional Notes)

- [x] Yes, I have added a new example under [examples](https://github.com/aws-samples/crossplane-aws-blueprints/tree/main/examples) to support my PR

- [ ] Yes, I have updated the [docs](https://github.com/aws-samples/crossplane-aws-blueprints/tree/main/doc) for this feature

- [ ] Yes, I have linked to an issue or feature request (applicable to PRs that solves a bug or a feature request)

**Note**:
 - Not all the PRs require examples and docs
 - We prefer small, well tested pull requests. Please ensure your pull requests are self-contained, and commits are squashed

### For Moderators
- [ ] E2E Test successfully complete before merge?

### CODE REVIEW

File: examples/gatekeeper/required-tags/template.yaml [hunk 1]
```diff
@@ -8,31 +8,18 @@ spec:
...
-        # Schema for the `parameters` field
         openAPIV3Schema:
           type: object
           properties:
-            tags:
-              type: array
-              items:
-                type: string
+            tag:
+              type: string
```
Comment (candonov) [inline examples/gatekeeper/required-tags/template.yaml:15]: why only one tag? it should be list of k/v pair of tags

File: examples/gatekeeper/required-tags/template.yaml [hunk 2]
```diff
@@ -8,31 +8,18 @@ spec:
...
   targets:
     - target: admission.k8s.gatekeeper.sh
       rego: |
         package awsrequiredtags
-        import future.keywords.every
```
Comment (candonov) [inline examples/gatekeeper/required-tags/template.yaml:23]: What does this do and why remove?

File: examples/gatekeeper/required-tags/samples/constraint.yaml [hunk 3]
```diff
@@ -5,7 +5,7 @@ metadata:
...
   match:
     kinds:
-      - apiGroups: ["*"]
-        kinds: ["*"]
+      - apiGroups: ["dynamodb.aws.upbound.io"]
+        kinds: ["Table"]
   parameters:
-    tags: ["owner"]
\ No newline at end of file
+    tag: "owner"
```
Comment (candonov) [inline examples/gatekeeper/required-tags/samples/constraint.yaml:11]: this should be tags and a list

File: examples/gatekeeper/required-tags/template.yaml (hunk 1 above)
Comment (7navyasa) [reply examples/gatekeeper/required-tags/template.yaml]: Fix pushed

File: examples/gatekeeper/required-tags/samples/constraint.yaml (hunk 3 above)
Comment (7navyasa) [reply examples/gatekeeper/required-tags/samples/constraint.yaml]: Fix pushed

**Review by candonov — APPROVED** [review]

**Pull request merged**